### PR TITLE
support new event formats of: hclog-text and hclog-json

### PR DESCRIPTION
**Pull request by jimlambrt**

### CODE REVIEW

File: internal/observability/event/node.go [hunk 1]
```diff
@@ -11,17 +11,17 @@ import (
 	"github.com/hashicorp/go-bexpr"
 )
 
-// Node represents an eventlogger.Node which filters events based on allow and
+// cloudEventsFormatterFilter represents an eventlogger.cloudEventsFormatterFilter which filters events based on allow and
```
Comment (jefferai) [inline internal/observability/event/node.go:14]: Does it make sense to rename the file based on the renaming inside it? Just for organizational purposes?

**Review by jefferai — APPROVED** [review]
10/10, would observe again

File: internal/observability/event/node.go (hunk 1 above)
Comment (jimlambrt) [reply internal/observability/event/node.go:14]: ty!

**Review by jefferai — APPROVED** [review]

File: internal/observability/event/hclog_formatter_node_test.go [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+				return
+			}
+			require.NoError(err)
+			assert.NotNil(got)
+
+			assert.Equal(tt.jsonFormat, got.jsonFormat)
+
+			assert.Len(got.allow, len(tt.wantAllow))
+			for _, f := range got.allow {
+				assert.Contains(tt.wantAllow, f.raw)
```
Comment (s-christoff) [inline internal/observability/event/hclog_formatter_node_test.go:360]: Should these be flipped (we are checking `f.raw` for something matching `tt.wantAllow`)?

**Review by s-christoff — APPROVED** [review]
one minor comment but 🤙

File: internal/observability/event/hclog_formatter_node_test.go (hunk 2 above)
Comment (jimlambrt) [reply internal/observability/event/hclog_formatter_node_test.go:360]: we're checking the list of all possible wants (a slice of strings) to see if a specific got string is present.

**Pull request merged**